web/router.ex

```diff
@@ -17,7 +17,7 @@ defmodule Notox.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
-    resources "/notes", NoteController, only: [:index]
+    resources "/notes", NoteController, only: [:index, :new, :edit, :create, :update]
```

Comment:
Needed to add `:create` and `:update` so that the `:new` and `:edit` templates would compile.